tools/sambamba_merge.cwl

```diff
@@ -0,0 +1,43 @@
+cwlVersion: v1.0
+class: CommandLineTool
+id: sambamba_merge
+
+requirements:
+  - class: DockerRequirement
+    dockerPull: 'kfdrc/sambamba:0.7.1'
+  - class: ShellCommandRequirement
+  - class: InlineJavascriptRequirement
+  - class: ResourceRequirement
+    ramMin: 32000
+    coresMin: 16
+baseCommand: ["/bin/bash", "-c"]
+arguments:
+  - position: 0
+    shellQuote: false
+    valueFrom: >-
+      set -eo pipefail
+      
+      sambamba merge
+      $(inputs.output_basename).bam
+      -t 16
+
+  - position: 3
+    shellQuote: false
+    valueFrom: >-
+      && mv $(inputs.output_basename).bam.bai $(inputs.output_basename).bai
+
+inputs:
+  input_bams:
+    type:
+      type: array
+      items: File
+      inputBinding:
+        separate: false
+        position: 2
```

Comment:
I might be wrong but I think this position argument needs to be at the top level. Also whats teh separate for?

Reply:
Eh, in another test run it performed as expected. Maybe I could get away with position 1.
Separate def: ```If true (default), then the prefix and value must be added as separate command line arguments; if false, prefix and value must be concatenated into a single command line argument.```